packages/core/src/dialects/snowflake/query-interface.js

```diff
@@ -1,38 +1,8 @@
 'use strict';
```

Comment:
This file can now be removed, right? Or if we still need it for whatever reason it can be migrated to TS now. There is already a .d.ts file for it

Reply:
Good point. I've removed the .d.ts file and migrated it to typescript.